sippy-ng/src/pull_requests/PullRequestsTable.js

```diff
@@ -178,9 +265,25 @@ export default function PullRequestsTable(props) {
         )
       },
     },
+    history: {
+      field: 'history',
+      headerName: 'History',
```

Comment:
On this one and the GitHub, if you click the sort arrow, the output gets messed up.  Not sure how to exclude those from the `updateSortModel` (assuming that's how you do it).

Reply:
It's just a matter of setting sortable to false for this column - https://mui.com/x/react-data-grid/sorting/#disable-the-sorting